src/ledger.ts

```diff
@@ -388,9 +356,10 @@ export abstract class LedgerBitcoinInteraction extends LedgerInteraction {
    * Inheriting classes should call the super.run()
    * as well as set the properties of support before calling their run
    * in order to check support before calling the actual interaction run
-   * @returns {Promise} could return void or some value that the interaction is meant to provide
+   *
+   * The return type has to remain any to get inheritance typing to work.
    */
-  async run(): Promise<unknown> {
+  async run(): Promise<any> {
```

Comment:
My understanding is that `unknown` is [preferable](https://www.benmvp.com/blog/when-use-typescript-unknown-versus-any/) over `any`

Reply:
I read the article, and I agree with it, but the article really isn't making the case that it's generally preferred over `any`. It describes an environment with strict type control and which you actually don't know the type of something. I think `unknown` is especially great when dealing with external APIs.

In this case, the type is actually known. `run` inherits an extensive union type. Rather than blowing up the scope to bring in a messy union, I figured we can leave it as `any` for now. Another helpful thing about `any` (for now), is that when we turn on `noExplicitAny` it will show us what we must fix.

Please consider this comment for many other cases where `any` was used instead of `unknown`. In most of them, changing to `unknown` wasn't so simple for this reason and others.

Reply:
Yeah so basically `any` is being used to document a type we want to update as a next step. Let’s just mention that in a dedicated comment somewhere like I mentioned elsewhere and then these are all fine with me.

Reply:
Readme note added in 8a7d8b3ff59d054336ed85b21940f79453a7f606